openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
+
+            possible_vote_lines = lines[8:]
+            for l in possible_vote_lines:
+                l = l.replace("\xc2\xa0"," -")
+                if "yea -" in l.lower():
+                    currently_counting = "yes_votes"
+                elif "nay -" in l.lower():
+                    currently_counting = "no_votes"
+                elif "excused -" in l.lower():
+                    currently_counting = "other_votes"
+                elif "not voting -" in l.lower():
+                    currently_counting = "other_votes"
+                elif currently_counting == "":
+                    pass
+                elif re.search(r'v\. \d\.\d',l):
+                    #this gets rid of the version number
+                    #which is often found at the bottom of the doc
+                    pass
+                else:
+                    voters = l.split("  ")
+                    for v in voters:
+                        if v.strip():
+                            vote[currently_counting].append(v.strip())
+
+            assert (len(vote["yes_votes"]) == vote["yes_count"],
+                "Yes vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["yes_count"],actual=len(vote["yes_votes"])))
+            assert (len(vote["no_votes"]) == vote["no_count"],
+                "No vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["no_count"],actual=len(vote["no_votes"])))
+            assert (len(vote["other_votes"]) == vote["other_count"],
+                "Other vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["other_count"],actual=len(vote["other_votes"])))
+            
+
+            #vote should have the majority if it passed
+            if vote["passed"] and vote["yes_count"] <= vote["no_count"]:
+                raise AssertionError("The vote should have at least a majority if it passed.")
+
+            #if vote has the majority and didn't pass, throw a warning
+            #this is possible in supermajority cases,
+            #but we might want to think about it if it starts happening lots
+            if not vote["passed"] and vote["yes_count"] > vote["no_count"]:
+                self.logger.warning("Vote got a majority but didn't pass. Probably worth a look.")
+
+
+            bill.add_vote(vote)
+
+    def deal_with_version(self,version,bill,proxy):
+        #documents
+        docs = {"Amendment":version["amendments"],
+                "Fiscal Note":version["fiscal-notes"],
+                "Committee Amendment":version["cmte_amendments"],
+                "Floor Amendment":version["floor_amendments"],
+                "Committee Report":version["committee-reports"]
+                }
+        for doc_type,doc_list in docs.items():
+            for doc in doc_list:
+                title = "{doc_type}: {name}".format(doc_type=doc_type,name=doc["name"])
+                url = proxy["url"] + doc["link"]
```

Comment:
I'm seeing some repeated URLs. For example, HB 1372 has the following document link twice, once as an `Amendment` and once as a `Committee Amendment`:
`http://in.proxy.openstates.org/2015/bills/hb1372/versions/hb1372.01.intr/amendments/HB1372.01.INTR.AMH01`

I'm not sure what de-duping strategy would work best; your instinct is probably right after spending all this time on Indiana!